osu.Game/Screens/Select/CarouselContainer.cs

```diff
@@ -248,6 +251,13 @@ protected override void Update()
             }
         }
 
+        public void InvalidateVisible()

```

Comment:
Not invalidate in the sense of Drawable.Invalidate, but it says "the Y positions of panels is now wrong, the Hidden state of some of them has changed"